cdap-distributions/bin/prep_and_stage_bundles.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+#
+# Copyright © 2014-2015 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+######################################################################################
+# Create RPM and Debian bundles and upload them directly to docs1
+# This script expects to receive 3 parameters in this order:
+#   $1 BUILD_PACKAGE    e.g. cdap
+#   $2 VERSION    e.g. 3.0.4
+#   $3 USER                     REMOTE_USER
+#   $4 remote host              REMOTE_HOST
+#   $5 target directory         REMOTE_INCOMING_DIR
+######################################################################################
+DEBUG=${DEBUG:-no}              ### set to yes for debugging
+RUN_DATE=`date '+%Y%m%d_%R'`
+SCRIPT=`basename ${BASH_SOURCE[0]}`                     ### Set Script Name variable
+BUILD_PACKAGE="${1:-cdap}"                                    ## e.g. cdap ## a.k.a. BUILD_PACKAGE
```

Comment:
Plus, this is in the CDAP repository, do it'll only ever be `cdap`... :smile: 

Since this is coming from local disk, there's no reason why we need to specify project and version. The project is always `cdap` and the version is whatever's on the disk.

```
VERSION=$(<cdap-distributions/target/stage-packaging/opt/cdap/distributions/VERSION)
```